Scripts/script-ReadPDFFile.yml

```diff
@@ -56,6 +57,9 @@ script: |-
           except PSEOF:
               mark_suspicious()
               sys.exit(0)
+          except PDFEncryptionError as err:
```

Comment:
Let's handle this similar to how we handle PSEOF and not return an error. Human readable content should contain the error message as a regular entry.

Reply:
Not sure if that's the right way to go here, when we encounter PSEOF we mark the file as suspicious and return an error entry, are you sure it's the right way to go here?